scripts/deploy.sh

```diff
@@ -122,8 +122,14 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
     popd
   fi
 
-  # setup the infrastructure
-  run_ansible setup-infrastructure.yml
+  # setup the infrastructure, run repo build to build untagged release
+  run_ansible memcached-install.yml
+  run_ansible repo-server.yml
+  run_ansible repo-build.yml
```

Comment:
This is a diversion from how QE/support execute the builds in production. Please ensure that this does not merge.